docs/demos/basics/index.rst

```diff
@@ -1,15 +1,30 @@
 StellarGraph basics
 ===================
 
-`StellarGraph <https://github.com/stellargraph/stellargraph>`_ has support for loading data via Pandas and NetworkX. This folder contains examples of the loading data into a ``StellarGraph`` object, which is the format used by the machine learning algorithms in this library.
+`StellarGraph <https://github.com/stellargraph/stellargraph>`_ has support for loading data via Pandas, NetworkX and Neo4j. This folder contains examples of the loading data into a ``StellarGraph`` object, which is the format used by the machine learning algorithms in this library.
 
 Find demos for a format
 -----------------------
 
-
-* :doc:`loading-pandas <loading-pandas>` shows the recommended way to load data, using Pandas (supporting any input format that Pandas supports, including CSV files and SQL databases)
-* :doc:`loading-networkx <loading-networkx>` shows how to load data from a `NetworkX <https://networkx.github.io>`_ graph
-* :doc:`loading-saving-neo4j <loading-saving-neo4j>` shows how to load data from a `Neo4j <https://neo4j.com>`_ database, and save results back to it
+.. list-table::
+   :header-rows: 1
+
+   * - Demo
+     - Data formats
+     - Performance
+     - Data preprocessing
+   * - :doc:`loading-pandas`
+     - Anything `supported by Pandas <https://pandas.pydata.org/pandas-docs/stable/user_guide/io.html>`__: CSV, TSV, Excel, JSON, SQL, HDF5, many more
+     - Good
+     - Via Pandas and `scikit-learn <http://scikit-learn.github.io/stable>`__ and more
```

Comment:
possibly a comma instead of and?  technically the demo doesn't show use of scikit-learn?

Reply:
I'm assuming you mean:

```suggestion
     - Via Pandas, `scikit-learn <http://scikit-learn.github.io/stable>`__ and more
```

Also, the demo does use `train_test_split` from scikit-learn in the Cora section. Arguably this isn't quite preprocessing, but I think it's close enough:

```python
from sklearn import model_selection

cora_train, cora_test = model_selection.train_test_split(
    cora_subject, train_size=0.25, random_state=123
)
cora_train
```